src/translations/en.json

```diff
@@ -8554,10 +8587,11 @@
       }
     },
     "tips": {
-      "key_c_hint": "Press 'c' on any page to open the command dialog",
-      "key_e_hint": "Press 'e' on any page to open the entity search dialog",
-      "key_m_hint": "Press 'm' on any page to get the My Home Assistant link",
-      "key_a_hint": "Press 'a' on any page to open the Assist dialog"
+      "keyboard_shortcut": "keyboard shortcut",
+      "key_c_hint": "Press {keyboard_shortcut} 'c' on any page to open the command dialog",
+      "key_e_hint": "Press {keyboard_shortcut} 'e' on any page to open the entity search dialog",
+      "key_m_hint": "Press {keyboard_shortcut} 'm' on any page to get the My Home Assistant link",
+      "key_a_hint": "Press {keyboard_shortcut} 'a' on any page to open the Assist dialog"
```

Comment:
We probably should change to new keys here, due to translations

Reply:
Yes, due to introducing "keyboard_shortcut" there.

Reply:
Thanks for confirming, will do 👍